rental.rb

```diff
@@ -0,0 +1,21 @@
+# Create a class Rental with the following:
+# @date instance variable, should be initialized in the constructor.
+# Setter and getter for @date (remember about attr_accessor).
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    # similar to books. Now, a person/book object knows which rentals are associated with them
+    @person = person
+    # person.rental << self
+
+    # add the current Rental object (self) to the list of rentals (rentals attribute) of the book object.
+    @book = book
+    # book.rentals << self
```

Comment:
Nice job taking away some of the commented pieces of code from your codebase as requested by the previous reviewer😉. but it seems you have forgotten some of them in your `rental.rb` file. kindly do well to check all your files and take them away as they are also present in some other files😉.